cmd/ui/src/views/Administration/Administration.tsx

```diff
@@ -48,6 +50,11 @@ const Administration: React.FC = () => {
                     path: ROUTE_ADMINISTRATION_DATA_QUALITY,
                     component: QA,
                 },
+                {
+                    label: 'Database Management',
```

Comment:
Did we want to keep this as Database Management ? 🤔 Probably should check with product

Reply:
yep! this is the name specified in the acceptance criteria